core/config/toml/types.go

```diff
@@ -1452,6 +1452,26 @@ func (r *ExternalRegistry) setFrom(f *ExternalRegistry) {
 	}
 }
 
+type WorkflowRegistry struct {
+	Address   *string
+	NetworkID *string
+	ChainID   *string
+}
+
+func (r *WorkflowRegistry) setFrom(f *WorkflowRegistry) {
```

Comment:
Would an unset or empty field invalidate the node's config on boot if the `WorkflowRegistry` is defined in the config but is missing a field or value?
Please reflect it (the invalid configuration) in your `test data/scripts` by showing what would be an invalid way of setting the config.

Reply:
No, these are optional values so they can be unset.